gwsurrogate/surrogate.py

```diff
@@ -196,18 +196,21 @@ def __call__(self, q, M=None, dist=None, phi_ref=None,\
     # at a given value of parameter space. These calibration parameters will scale the 
     # time and amplitude. Currently, these models use calibration:
     # BHPTNRSur1dq1e4, EMRISur1dq1e4
-    if(self.surrogateID == 'BHPTNRSur1dq1e4'):
+    if not calibrated:
```

Comment:
perhaps nest this entire if-statement under `if(self.surrogateID in ['EMRISur1dq1e4','BHPTNRSur1dq1e4']):`. So it could be something like

```python
if(self.surrogateID in ['EMRISur1dq1e4','BHPTNRSur1dq1e4']):
  # CODE YOU ALREADY HAVE
else: 
   alpha_nr_calibration, beta_nr_calibration = None, None
```

Which is again to be extra careful about not defining calibration parameter values for models that don't use them